api/CredentialsContainer.json

```diff
@@ -155,7 +155,8 @@
                 "samsunginternet_android": "mirror",
                 "webview_android": {
                   "version_added": false
-                }
+                },
+                "webview_ios": "mirror"
```

Comment:
For the record: It would be plausible if CredentialsContainer is not supported by Webview iOS, given that it's available in Chrome Android, but _not_ in Webview Android.

However, I didn't find any mention of ["webview" in the WebKit credential management source code](https://github.com/search?q=repo%3AWebKit%2FWebKit%20path%3A%2F%5ESource%5C%2FWebCore%5C%2FModules%5C%2Fcredentialmanagement%5C%2F%2F%20webview&type=code), so `"mirror"` is a good guess.